src/main/java/picard/analysis/directed/RnaSeqMetricsCollector.java

```diff
@@ -380,7 +380,10 @@ else if (this.metrics.SAMPLE != null) {
 
                     fivePrimeSkews.increment(fivePrimeCoverage / mean);
                     threePrimeSkews.increment(threePrimeCoverage / mean);
-                    fiveToThreeSkews.increment(fivePrimeCoverage / threePrimeCoverage);
+                    final double fiveToThree = (Math.abs(((double)0.0) - threePrimeCoverage) >  (double).000001)
+                            ? fivePrimeCoverage / threePrimeCoverage
+                            : (double)0.0;
+                    fiveToThreeSkews.increment(fiveToThree);
```

Comment:
What's this PO issue you're referring to ;)?  It may be nice to add a "divide" method to `MathUtil` taht does this and then you can use it elsewhere (lines 381-382).